### PR TITLE
Add Ada4Science BOSON pool and their first donation

### PR DESCRIPTION
**Pull request by SOERGI**

### CODE REVIEW

File: BOSON/donations.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    name: Cybathlon, Reha-Initiative of the ETH Zurich Foundation
+    address: Weinbergstrasse 29, 8006 Zurich, Switzerland
+    tel: +41 44 632 49 08
+    contact: spenden@ethz-foundation.ch
+  ada: 90
+  usd: 108.7
+  timestamp: 1627333440 
+  files:
+    - BOSON/epoch_278_cybathlon_letter_small.jpg
+    - BOSON/epoch_278_cybathlon.png
```
Comment (kiriakos) [inline BOSON/donations.yml:14]: Hey there! Thanks for your submission. The proofs and yml are looking good but I have one question. How come you blanked out the time component on the top right of this screenshot?

Comment (SOERGI) [reply BOSON/donations.yml:14]: Geia sou Kiriako!
That's just me being overly careful what information I leak into the internet. I created the screenshot for our webpage before I submitted the images here and just reused them. I can take a new screenshot if you want and update the PR, no problem.

Comment (kiriakos) [reply BOSON/donations.yml:14]: No trouble, I'm fine with merging it right now but consider this for the future OK?

**Review by kiriakos — APPROVED** [review]
LGTM

File: BOSON/donations.yml (hunk 1 above)
Comment (SOERGI) [reply BOSON/donations.yml:14]: Sure thing, thanks!

**Pull request merged**